docs/guides/migration.md

```diff
@@ -0,0 +1,223 @@
+
+2. **Verify the Changes**: After the apply completes, verify that the changes have been applied correctly and that your environment is functioning as expected. The state file should now reflect the new attributes.
+
+### Step 5: Migrate deprecated configuration
+
+1. **Identify the deprecated attributes**: The `terraform plan` command only displays a single warning, which can make it hard to do a full analysis of which attributes are deprecated.
+
+    ```bash
+    No changes. Your infrastructure matches the configuration.
+
+    Terraform has compared your real infrastructure against your configuration and found no differences, so no changes are needed.
+    ╷
+    │ Warning: Attribute Deprecated
+    │ 
+    │   with aci_bridge_domain.terraform_bd,
+    │   on main.tf line 70, in resource "aci_bridge_domain" "terraform_bd":
+    │   70:   tenant_dn = aci_tenant.terraform_tenant.id
+    │ 
+    │ Attribute 'tenant_dn' is deprecated, please refer to 'parent_dn' instead. The attribute will be removed in the next major version of the provider.
+    │ 
+    │ (and 3 more similar warnings elsewhere)
+    ```
+
+    To display the other warnings the `terraform validate` command can be used.
+
+    ```bash
+    terraform validate -json | jq '.diagnostics[] | {detail: .detail, filename: .range.filename, start_line: .range.start.line}'
+    {
+        "detail": "Attribute 'unk_mcast_act' is deprecated, please refer to 'l3_unknown_multicast_flooding' instead. The attribute will be removed in the next major version of the provider.",
+        "filename": "main.tf",
+        "start_line": 73
+    }
+    {
+        "detail": "Attribute 'll_addr' is deprecated, please refer to 'link_local_ipv6_address' instead. The attribute will be removed in the next major version of the provider.",
+        "filename": "main.tf",
+        "start_line": 72
+    }
+    ```
+
+2. **Change the deprecated attributes**: Replace the deprecated attributes in your configuration file with the new attributes and execute the plan again.
+
+    ***Old Configuration***
+    ```hcl
+    resource "aci_bridge_domain" "terraform_bd" {
+        tenant_dn = aci_tenant.terraform_tenant.id
+        name = "terraform_bd"
+        ll_addr = "::"
+        unk_mcast_act = "flood"
+    }
+    ```
+
+    ***New Configuration***
+    ```hcl
+    resource "aci_bridge_domain" "terraform_bd" {
+        parent_dn = aci_tenant.terraform_tenant.id
+        name = "terraform_bd"
+        link_local_ipv6_address = "::"
+        l3_unknown_multicast_flooding = "flood"
+    }
+    ```
+
+    The `terraform plan` command should not display any warnings anymore.
+
+    ```bash
+    No changes. Your infrastructure matches the configuration.
+
+    Terraform has compared your real infrastructure against your configuration and found no differences, so no changes are needed.
+    ```
+
+### Step 6: Cleanup
+
+1. **Remove Backup (Optional)**: If everything is working correctly, you can remove the backup state file.
```

Comment:
Does the colour of the font change? There are a few inconsistencies

Reply:
What do you mean by this? The colours displayed here in GitHub? 

I used the https://registry.terraform.io/tools/doc-preview tool and VsCode, here all the titles and bullet points seems consistent